app/views/home/index.html.erb

```diff
@@ -0,0 +1,8 @@
+<div class="home-container">
+  <h1 class="app-name">Budget Buddy</h1>
+  <div class="btn-wrapper">
+    <%= link_to "Login", new_user_session_path, class: "login-btn" %>
+    <%= link_to "Sign Up", new_user_registration_path, class: "sign-up-btn" %>
+    <%#= button_to 'Log out', destroy_user_session_path, method: :delete, class: "btn" %>
+  </div>
+</div>
```

Comment:
Nice job so far😉. Let's try to stick to the design both in color, typography, positioning and sizes of fonts and elements as this is a requirement that we stick to the given design😉.

yours
<img width="438" alt="image" src="https://github.com/anita00001/budget-app/assets/64643995/75e145ed-4e4a-4eca-b0db-f8b9c4c74aa8">

design
<img width="232" alt="image" src="https://github.com/anita00001/budget-app/assets/64643995/89c30fd7-989f-4bde-81f8-a38a5b03aec1">

Reply:
Thanks for your suggestion. I have made necessary changes.